sickbeard/versionChecker.py

```diff
@@ -185,12 +185,29 @@ def showupdate_safe(self):
             else:
                 logger.log(u"We can't proceed with the update. Shows are being updated", logger.DEBUG)
                 return False
+                
+        def showqueue_empty(self):
+            queuelength = sickbeard.searchQueueScheduler.action.queue_length()
+            failed = int(queuelength['failed'])
+            manual = int(queuelength['manual'])
+            daily = int(queuelength['daily'])
+            backlog = int(queuelength['backlog'])
+            action_size = len(sickbeard.showQueueScheduler.action.queue)
+            queue = failed + manual + daily + backlog + action_size
+            if  queue == 0:
+                logger.log(u"We can proceed with the update. Show queue is empty", logger.DEBUG)
```

Comment:
Do we need to log something if the update can be performed? Don't we already log something when the update starts?